docs/sources/helm-charts/mimir-distributed/run-production-environment-with-helm/monitor-system-health.md

```diff
@@ -138,3 +138,8 @@ metaMonitoring:
           passwordSecretName: gem-tokens
           passwordSecretKey: metamonitoring
 ```
+
+{{% docs/reference %}}
+[Installing Grafana Mimir dashboards and alerts]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/monitor-grafana-mimir/installing-dashboards-and-alerts"
+[Collect metrics and logs without the Helm chart]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/monitor-grafana-mimir/collecting-metrics-and-logs#collect-metrics-and-logs-without-the-helm-chart"
```

Comment:
```suggestion
[Installing Grafana Mimir dashboards and alerts]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/monitor-grafana-mimir/installing-dashboards-and-alerts"
[Collect metrics and logs without the Helm chart]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/monitor-grafana-mimir/collecting-metrics-and-logs#collect-metrics-and-logs-without-the-helm-chart"
```

Reply:
This suggestion is not correct for the release of the Mimir documentation targeted by the Helm chart documentation. There is no "Manage" section in the `v2.9.x` documentation (https://grafana.com/docs/mimir/v2.9.x/).

Reply:
This PR looks like it is opened against `main`.

Reply:
Oh! You are handing the version with a parameter instead of a CI/CD label. That threw me off.

Reply:
I'll need someone more familiar with the Mimir Helm chart release process to ensure that this is targeting the correct branch and also how we get the updates where they need to go to update the Mimir Helm chart latest documentation.

Presently that documentation has broken links (like [the https://grafana.com/docs/helm-charts/mimir-distributed/latest/run-production-environment-with-helm/monitor-system-health/#monitor-the-health-of-your-system](https://grafana.com/docs/helm-charts/mimir-distributed/latest/run-production-environment-with-helm/monitor-system-health/#monitor-the-health-of-your-system:~:text=Collect%20metrics%20and%20logs%20without%20the%20Helm%20chart) that work correctly in this branch.

Reply:
I have added Krajo because he has context about the release process for Helm chart documentation and was the one to parameterize this initially.